bot/exts/moderation/infraction/infractions.py

```diff
@@ -444,43 +463,51 @@ async def pardon_ban(self, user_id: int, guild: discord.Guild, reason: t.Optiona
 
         return log_text
 
-    async def pardon_voice_ban(self, user_id: int, guild: discord.Guild, reason: t.Optional[str]) -> t.Dict[str, str]:
-        """Add Voice Verified role back to user, DM them a notification, and return a log dict."""
+    async def pardon_voice_ban(
+        self,
+        user_id: int,
+        guild: discord.Guild,
+        *,
+        notify: bool = True
+    ) -> t.Dict[str, str]:
+        """Optionally DM the user a pardon notification and return a log dict."""
```

Comment:
Did you mean to overwrite the whole docstring here?

Reply:
Yes, because it was incorrect: the function doesn't actually add the role back, and this was confirmed to be intentional behaviour.